code/game/gamemodes/game_mode.dm

```diff
@@ -293,25 +293,33 @@
 		intercepttext += "<hr>"
 		intercepttext += report
 
-	if(station_goals.len)
-		intercepttext += "<hr><b>Special Orders for [station_name()]:</b>"
-		for(var/datum/station_goal/G in station_goals)
-			G.on_report()
-			intercepttext += G.get_report()
-
-	if(SSstation.station_traits.len)
-		intercepttext += "<hr><b>Identified shift divergencies:</b>"
-		for(var/i in SSstation.station_traits)
-			var/datum/station_trait/station_trait_iterator = i
-			if(!station_trait_iterator.show_in_report)
-				return
```

Comment:
Turns out for normal gamemodes, the fix for station traits blocking reports was never mirrored over 🥴